### PR TITLE
[MIG] mail_layout_preview: Migration to 14.0

### PR DESCRIPTION
**Pull request by phuctranfxvn**

### CODE REVIEW

**Comment by dreispt** [comment]
@OCA/social-maintainers Can we have a review please?

**Comment by simahawk** [comment]
on it ;)

**Review by dreispt — APPROVED** [review]

File: mail_layout_preview/tests/test_layout_preview.py [hunk 1]
```diff
@@ -16,7 +16,7 @@ def _create_template(env, model, **kw):
             "subject": "Preview ${object.name}",
             "body_html": "<p>Hello ${object.name}</p>",
             "model_id": env["ir.model"]._get(model).id,
-            "user_signature": False,
+            # "user_signature": False,
```
Comment (simahawk) [inline mail_layout_preview/tests/test_layout_preview.py]: why this?

**Review by simahawk — APPROVED** [review]
I'm glad I wrote tests :sweat_smile: 
Just a little comment/question!

File: mail_layout_preview/tests/test_layout_preview.py (hunk 1 above)
Comment (phuctranfxvn) [reply mail_layout_preview/tests/test_layout_preview.py]: user_signature has been removed in 14.0 and it is useless for the test, I think

Comment (simahawk) [reply mail_layout_preview/tests/test_layout_preview.py]: just delete it then ;)

Comment (phuctranfxvn) [reply mail_layout_preview/tests/test_layout_preview.py]: updated, thanks

**Pull request merged**